config/routes.rb

```diff
@@ -30,23 +30,21 @@
 
   resources :custom_manifests, only: [:create, :update, :destroy, :show]
 
-  scope "user" do
-    resources :counters, controller: "user_counters", only: [:update] # Legacy route for GodTools Android v5.7.0-v6.0.0
-
-    scope "me" do
-      resources :counters, controller: "user_counters", only: [:index, :update]
-    end
-  end
+  patch "user/counters/:id", to: "user_counters#update"    # Legacy route for GodTools Android v5.7.0-v6.0.0
```

Comment:
Rubocop is being even more pedantic than me :-/ - doesn't like the comment spacing